src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,22 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE_FOR_CANDIDATES = 35;
```

Comment:
```suggestion
    private static final int MIN_AGE = 35;
```
This can be so unspecified, as it is the only constant for minimum age